src/libs/routes/errorHandler.ts

```diff
@@ -0,0 +1,13 @@
+const var_err =  {
```

Comment:
- It will be in errorHandler method
- error, message, status will be destructured from err parameter
- default error will be 'Not Found' if nothing is coming inside err parameter